refinery/tool_manager/tests.py

```diff
@@ -869,6 +868,14 @@ def test_get_extra_directories_workflow_tool_def(self):
         with self.assertRaises(NotImplementedError):
             self.td.get_extra_directories()
 
+    def test_related_workflow_inactive_after_deletion(self):
+        self.create_workflow_tool_definition()
+        self.assertTrue(self.td.workflow.is_active)
+        self.td.delete()
+        self.assertFalse(
+            Workflow.objects.all()[0].is_active
+        )
```

Comment:
Would you like to make an assertion about `self.td` here? It's not immediately clear what the relationship to `Workflow.objects.all()[0]` would be...